3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -74,7 +74,7 @@
         ).resolve(strict=True)),
         "dest_path": str(pathlib.Path(f"{output_dir}/{name}.parquet")),
     }
-    for name in plate_names if name in ["Plate_3", "Plate_3_prime", "Plate_4"]  # focus on Plate_3, Plate_3_prime, and Plate_4
+    for name in plate_names if not name in ["Plate_1", "Plate_2"]  # focus on non-pilot plates
```

Comment:
You repeat this pattern (`["Plate_1", "Plate_2"]`) below in line 120 - you might consider defining earlier and referencing. E.g.:

```python
pilot_plates = ["Plate_1", "Plate_2"]
```

Reply:
Good point! I will implement this!